### ISSUE TITLE
Adding the VQACPv2 dataset

### ISSUE THREAD

**Issue by gkv91** [facebookresearch/mmf#426]
## 🚀 Feature

Provide testing the VQA performance on VQACPv2
## Motivation
VQACPv2 can be used for testing the generalisability of VQA models. Hence, it is worth including.

**Comment by vedanuj** [comment]
Thanks! Yes this can be useful. Do you want to open up a PR for this?

**Comment by parthsuresh** [comment]
Hi @vedanuj I can work on this during the weekend if that's fine. I found the dataset [here](https://www.cc.gatech.edu/grads/a/aagrawal307/vqa-cp/), hope it's the right one. I found this PR #420 which implements the OKVQA dataset, would it be right to use it as a reference? 

I would also appreciate any other suggestions/pointers from your side.

**Comment by vedanuj** [comment]
Hi @parthsuresh, thanks for willing to take this up. 

Yes the link to the dataset is correct. The OKVQA dataset implementation is a good starting point. Let us know if you get stuck anywhere.

**Comment by parthsuresh** [comment]
Hi @vedanuj , I've submitted PR #448 that adds an implementation for the VQACPv2 dataset, along with a MMBT config for testing.

Just one question: 

The VQACPv2 only has training and test sets, see [here](https://www.cc.gatech.edu/grads/a/aagrawal307/vqa-cp/). Currently in my implementation, I've made the validation set and test set the same, just to see whether the implementation is right. Should I split the VQACPv2 training set into train and val sets and host them somwhere?
